sst/mtm1m3/Identify_Oscillations_Nightly.ipynb

```diff
@@ -0,0 +1,754 @@
+{
```

Comment:
<p>I ran this in USDF and got an error saying that this variable is note defined. Please, either delete this cell or move it down.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/reports-performance-summary/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Fixed</p>